src/odbc/unittests/common.c

```diff
@@ -809,6 +809,9 @@ odbc_mark_sockets_opened(void)
 TDS_SYS_SOCKET
 odbc_find_last_socket(void)
 {
+#ifdef TDS_NO_DM
+	return tds_get_s(((TDS_DBC*)odbc_conn)->tds_socket);
```

Comment:
This code is already there now, just some couple of lines below with an additional check

Reply:
So it is; sorry for missing that!